docs/source/conf.py

```diff
@@ -57,7 +57,7 @@
 
 # General information about the project.
 current_year = datetime.datetime.now().year
-project = 'Civis Client'
+project = 'Civis API Python Client'
```

Comment:
After this PR in the compiled docs:
<img width="223" alt="Screen Shot 2021-12-02 at 9 11 49 PM" src="https://user-images.githubusercontent.com/86482098/144619611-244eaf42-5bfb-4f93-9c91-5ed07e9d73e9.png">